plugins/cilium-docker/driver/driver.go

```diff
@@ -271,7 +276,7 @@ func (driver *driver) createEndpoint(w http.ResponseWriter, r *http.Request) {
 	//				sendError(w, "Unable to decode JSON payload: "+err.Error(), http.StatusBadRequest)
 	//				return
 	//			}
-	//			log.Debugf("PortBinding: %+v", &portmap)
+	//			log.WithField(logfields.Object, logfields.Repr(portmap)).Debug("PortBinding:")
```

Comment:
Maintaining comments - this is dedication to having good logs!

Reply:
or I was so mechanical with the find-replace that I didn't even notice ;)